packages/core/src/api-wrappers/defaultApis.ts

```diff
@@ -96,7 +96,11 @@ export const defaultApis = [
       oauthRequestApi: oauthRequestApiRef,
     },
     factory: ({ discoveryApi, oauthRequestApi }) =>
-      GithubAuth.create({ discoveryApi, oauthRequestApi }),
+      GithubAuth.create({
+        discoveryApi,
+        oauthRequestApi,
+        defaultScopes: ['read:user', 'repo'],
```

Comment:
Makes sense to highlight the possibility of supplying scopes here, but let's keep the existing defaults